plugins/catalog-backend/src/search/DefaultCatalogCollator.ts

```diff
@@ -81,6 +81,24 @@ export class DefaultCatalogCollator implements DocumentCollator {
     return formatted.toLowerCase();
   }
 
+  private isUserEntity(entity: Entity): entity is UserEntity {
+    return entity.kind.toUpperCase() === 'USER';
```

Comment:
Oh, you may want to [toLocaleUppercase('en-US')](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/toLocaleUpperCase), here @ainhoaL

Reply:
:o good point!! Thank you for catching that